satellite/payments/stripecoinpayments/invoices.go

```diff
@@ -43,7 +43,6 @@ func (invoices *invoices) List(ctx context.Context, userID uuid.UUID) (invoicesL
 			Amount:      stripeInvoice.AmountDue,
 			Status:      string(stripeInvoice.Status),
 			Link:        stripeInvoice.InvoicePDF,
-			End:         time.Unix(stripeInvoice.PeriodEnd, 0),
```

Comment:
At least from what Jens and me found, this only shows the time at which the generation process of the invoice on stripe side finished.